Dockerfile

```diff
@@ -12,6 +12,7 @@ WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
 
+RUN apt-get update && apt-get install -y libgif-dev
```

Comment:
This should go before the package.json `COPY` instructions, so that this build command will be cached even if npm dependencies change.

Reply:
the game